tests/testdata/modules/site-health/audit-autoloaded-options/class-autoloaded-options-set.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * @package performance-lab
+ * @since 1.0.0
+ */
+
+/**
+ * Class Autoloaded_Options_Set sets extra options.
+ *
+ * @since 1.0.0
+ */
+class Autoloaded_Options_Set {
```

Comment:
This should be remove it a method on `Audit_Autoloaded_Options_Tests` class. No need to new class here.

Reply:
ok, agree